src/pages/Contact.vue

```diff
@@ -1,17 +1,195 @@
 <template>
-  <main>
-    <h1>Contact Page</h1>
-  </main>
+    <main>
+        <div class="contact-section"> 
+            <h1>Contact</h1>
+            <div class="contact-people">
+                <div class="contact-person">
+                    <h2>Lance Lao</h2>
+                    <span>Events Lead</span>
+                    <p>christopher.lao@obf.ateneo.edu</p>
```

Comment:
change this to a `a href="mailto:christopher.lao@obf.ateneo.edu">christopher.lao@obf.ateneo.edu</a>`